reference/features/filtering_and_faceted_search.md

```diff
@@ -0,0 +1,318 @@
+
+```
+[["genres = horror", "genres = comedy"], "director = 'Jordan Peele'"]
+```
+
+#### Combining arrays and strings
+
+You can also create filter expressions that use both the array and the string syntax.
+
+The following filter is written as a string and only returns movies not directed by `Jordan Peele` that belong to the `comedy` or `horror` genres:
+
+```
+"(genres = comedy OR genres = horror) AND director != 'Jordan Peele'"
+```
+
+You can write the same filter using mixing arrays and strings:
+
+```
+[["genres = comedy, genres = horror"], "NOT director = 'Jordan Peele'"]
+```
+
+### Example
+
+Suppose that you have dataset containing several movies in the following format:
+
+```json
+[
+    …
+    {
+        "id": "458723",
+        "title": "Us",
+        "director": "Jordan Peele",
+        "poster": "https://image.tmdb.org/t/p/w1280/ux2dU1jQ2ACIMShzB3yP93Udpzc.jpg",
+        "overview": "Husband and wife Gabe and Adelaide Wilson take their…",
+        "release_date": 1552521600,
+        "genres": [
+            "Comedy",
+            "Horror",
+            "Thriller"
+        ],
+        "rating": 4
+    },
+    …
+]
+```
+
+If you want to enable filtering using `director`, `release_date`, `genres`, and `rating`, you must add these attributes to the [`filterableAttributes` index setting](/reference/api/filterable_attributes.md).
+
+You can then restrict a search so it only returns movies released after 18 March 1995 with the following filter containing a single condition:
+
+```SQL
+release_date > 795484800
+```
+
+You can use this filter when searching for recent `Avengers` movies:
+
+<CodeSamples id="filtering_guide_1" />
+
+You can also combine multiple conditions. For instance, you can limit your search so it only includes recent movies directed by either `Tim Burton` or `Christopher Nolan`:
+
+```SQL
+release_date > 795484800 AND (director = "Tim Burton" OR director = "Christopher Nolan")
+```
+
+Here, the parentheses are mandatory: without them, the filter would return movies directed by `Tim Burton` and released after 1995 or any film directed by `Christopher Nolan`, without constraints on its release date. This happens because `AND` takes precedence over `OR`.
+
+You can use this filter when searching for `Batman` movies:
+
+<CodeSamples id="filtering_guide_2" />
+
+Note that filtering on string values is case-insensitive.
+
+If you only want well-rated movies that weren't directed by `Tim Burton`, you can use this filter:
+
+```SQL
+rating >= 3 AND (NOT director = "Tim Burton")
+```
+
+Querying on `Planet of the Apes`, the above example results in the following command:
+
+<CodeSamples id="filtering_guide_4" />
+
+## Faceted search
+
+**Faceted search** is a feature provided out-of-the-box by MeiliSearch. Faceting allows classifying search results into categories or **facets**. Facets are essentially a type of filter.
```

Comment:
```suggestion
**Faceted search** is a sub-filter provided by the filter feature. It was briefly discussed in the previous category. As it is a common feature used in search, the following will explain in depth what it is and how to use it.

Faceting allows classifying search results into categories or **facets**. Facets are essentially a type of filter.

example: In my clothes dataset I want all paints that are either bleu or yellow.
```

Reply:
Coming from my attempt to clarify that facets are a specific usage of filters, based mostly on how filter conditions/criteria are displayed:

```suggestion
MeiliSearch filters can be used to build **faceted search** interfaces, a specific type of interface that allows a user to refine search results based on broad categories or **facets**. For example, a clothing webshop can use faceted search to allow users to easily explore items in a specific color or belonging to a specific brand.
```